### PR TITLE
docs: update GEDI L4B V2.1 Summary

### PR DESCRIPTION
**Pull request by jjfrench**

### CODE REVIEW

File: products/GEDI_L4B/summary.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+  The Global Ecosystem Dynamics Investigation (GEDI) L4B product provides 1 km estimates of mean aboveground biomass density (AGBD). NASA's <a href="https://gedi.umd.edu/">GEDI</a> mission is a full waveform lidar instrument aboard the <a href="https://www.nasa.gov/mission_pages/station/main/index.html">ISS</a>. It launched in December 2018 and has been collecting 3D Earth surface and vegetation samples of temperate and tropical forests since April, 2019. <a href="https://www.sciencedirect.com/science/article/pii/S2666017220300018">GEDI is the first satellite lidar designed specifically to measure vegetation structure</a> and biomass, filling a critical data gap. Each laser illuminates ~25 m on the ground, and measures tree heights and volumes within those 25 m areas. <a href="https://daac.ornl.gov/GEDI/guides/GEDI_L4B_Gridded_Biomass_V2_1.html">GEDI estimates biomass</a> by relating lidar measurements to biomass collected in forest plots across the globe.
```
[the remark this answers is not in the document]
Comment (jjfrench) [reply products/GEDI_L4B/summary.html:2]: ah, I'll change this back

**Comment by nehahunka** [comment]
@lauraduncanson, I think everything on the dashboard regarding GEDI v2 has to be updated to v2.1...this includes the description on the CEOS web page (https://ceos.org/gst/GEDI-biomass.html). Who prepared this text, who can we ask to update it?

**Comment by lauraduncanson** [comment]
Stephen Ward ! And yes, agreedSent from my iPhoneOn Nov 12, 2023, at 1:58 PM, nehahunka ***@***.***> wrote:﻿
@lauraduncanson, I think everything on the dashboard regarding GEDI v2 has to be updated to v2.1...this includes the description on the CEOS web page (https://ceos.org/gst/GEDI-biomass.html). Who prepared this text, who can we ask to update it?

**Comment by jjfrench** [comment]
@nehahunka @lauraduncanson, if the summary looks okay then we can merge - or if you have comments/suggestions then feel free to add them here, thanks!

**Review by wildintellect — APPROVED** [review]

**Pull request merged**